pkg/auth/providers/saml/saml_provider.go

```diff
@@ -29,6 +29,7 @@ import (
 const PingName = "ping"
 const ADFSName = "adfs"
 const KeyCloakName = "keycloak"
+const OkTAName = "okta"
```

Comment:
nit: can we make the 'k' capital as well? `OKTAName`

Reply:
Year, of course, maybe I missed this problem, thanks!